### PR TITLE
Exercise layout by raul valencia

### PR DESCRIPTION
**Pull request by DkSanjed**
New exercise

### CODE REVIEW

File: 7-layouts/style.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+.sections_games {
+    display: grid;
+    grid-template: 300px 300px 300px/1fr 1fr 1fr;
+}
+
+@media screen and (max-width: 600px) and (min-width: 320px) {
+    h3 {
+        width: 96%;
```
Comment (juancho11gm) [inline 7-layouts/style.css]:
```suggestion
        width: 100%;
```

**Review by juancho11gm — CHANGES_REQUESTED** [review]
Very nice Raúl!. Please see the comments and sort the rules (A-z).
image

File: 7-layouts/style.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
+body {
+    color: whitesmoke;
+    margin: 0;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+footer {
```
Comment (pushdev-code) [inline 7-layouts/style.css]: always prefer classnames over tag names so you avoid side effects and forcing your markup to match your styles.

File: 7-layouts/style.css [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    background-color: rgb(39, 39, 39);
```
Comment (pushdev-code) [inline 7-layouts/style.css]:
```suggestion
    background-color: #272727;
```
If you don't use rgba + the opacity value, use hexadecimal for colors.

File: 7-layouts/style.css [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    color: aqua;
+    font-size: 10vw;
+    font-weight: 400;
+    margin: 0;
+    text-align: center;
+}
+
+h3 {
+    align-self: flex-end;
+    background-color: rgb(255, 255, 255, 0.25);
```
Comment (pushdev-code) [inline 7-layouts/style.css]:
```suggestion
    background-color: rgb(255, 255, 255, .25);
```
0 is always nothing, so you save 1 character by doing `.25`

**Review by pushdev-code — CHANGES_REQUESTED** [review]

**Pull request merged**